doc/api/esm.md

```diff
@@ -358,11 +358,11 @@ a namespace with a `default` export key pointing to the CommonJS
 `module.exports` value.
 
 This Module Namespace Exotic Object can be directly observed either when using
-`import * as m from 'cjs'` or a dynamic import:
+`import * as m from 'node:cjs'` or a dynamic import:
```

Comment:
Same here, we will need to determine whether this makes sense in this part of the docs.